spec/snapshots/Kitchen_Directions_BakeToc_supports_unit_numbering.snap

```diff
@@ -78,16 +81,18 @@
 <li cnx-archive-uri="" cnx-archive-shortid="" class="os-toc-unit" data-toc-type="unit">
   <a href="#">
     <span class="os-number"><span class="os-part-text">Unit </span>2</span>
-    <span class="os-divider"> </span>
-    <span data-type="" itemprop="" class="os-text">Unit 2 Title</span>
+<span class="os-divider"> </span>
+<span data-type="" itemprop="" class="os-text">Unit 2 Title</span>
+
   </a>
   <ol class="os-unit">
     
     <li class="os-toc-chapter" cnx-archive-shortid="" cnx-archive-uri="" data-toc-type="chapter">
   <a href="#chapTitle3">
     <span class="os-number"><span class="os-part-text">Chapter </span>2.1</span>
-    <span class="os-divider"> </span>
-    <span class="os-text" data-type="" itemprop="">Chapter 3 Title</span>
+<span class="os-divider"> </span>
+<span class="os-text" data-type="" itemprop="">Chapter 3 Title</span>
```

Comment:
any reason the indentation gets messed up in these snaps?

Reply:
Yes, it's related to adding the overrides to bake_toc. I didn't look into it very deeply because I figured indentation shouldn't matter. I will look into it a little more to see if I can match the indentation for the sake of trying to prevent differences.

Reply:
I tried matching indentation and that created a bunch of diffs in the recipe specs. I am more worried about those snapshots changing, so I am going to revert this change.

The fact that it wasn't creating diffs in the recipe specs before probably means it doesn't cause any meaningful changes in indentation in the final build, just in the snapshot for bake_toc.